core/src/edu/cornell/gdiac/temporary/entity/BandMember.java

```diff
@@ -8,6 +8,9 @@
 import edu.cornell.gdiac.temporary.*;
 import edu.cornell.gdiac.util.FilmStrip;
 
+import java.util.Map;
+import java.util.PriorityQueue;
```

Comment:
Do not use Java util data structures because I think they affect performance. Use LibGDX ones instead https://libgdx.com/wiki/utils/collections

(I think there is a queue also that they did not put here)